internal/pkg/release/version.go

```diff
@@ -62,6 +66,12 @@ func Snapshot() bool {
 	return err == nil && val
 }
 
+// Complete returns true if image was built as complete.
+func Complete() bool {
+	isComplete, ok := os.LookupEnv(complete)
+	return ok && isComplete == "true"
+}
+
 // VersionInfo is structure used by `version --yaml`.
 type VersionInfo struct {
```

Comment:
I don't think it would make sense to include it from the synthetics pov. For us, this is telling us that a given agent is running inside Docker with synthetics deps bundled. For us having `ELASTIC_AGENT_COMPLETE: "true"` on the diagnostics file is a direct correlation so no need to display it on the version. But I can add it if you think it'll add value

Reply:
Diagnostics includes the agent environment:
```
env:
    BEAT_SETUID_AS: elastic-agent
    ELASTIC_AGENT_COMPLETE: "true"
```